src/helpers/routes.js

```diff
@@ -24,6 +24,11 @@ exports.makeRoutes = (baseRoutes, {
     const routes = []
     let pageOptions
 
+    // Skip route if it is only a redirect.
+    if (route.redirect != null && !route.component) {
```

Comment:
We can't exclude an empty string because that is a valid redirect to the root of the site.

I chose `!= null` because it will catch only `null` and `undefined`. (I checked the lodash source and this is what _.isNil does.)

We could be explicit and say `(route.redirect !== null || route.redirect !== undefined) && !route.component`, but that seems less clear to me. I don't see another option unless we want to do `['string', 'object', 'function'].includes(typeof route.redirect)` which seems verbose and brittle to future vue-router changes.

Reply:
I'd prefer to be explicit. You could create a simple helper function (like lodash's) for it to make it clearer. I just don't tolerate loose comparisons.

Reply:
I tested vue-router and found that it doesn't actually support empty redirect paths so I simplified this to your original suggestion of `!route.redirect`.